app.js

```diff
@@ -0,0 +1,9 @@
+const { toggle } = require('cli-spinners');
+
+function dropFunction() {
+  document.getElementById('drop-content').classList.toggle('show');
+}
+dropFunction();
+toggle.addEventListener('click', () => {
+  toggle.classList.dropFunction();
+});
```

Comment:
- [ ] Great job so far but I think you have one thing left to make, after clicking on the three links the menu should close as well according to the requirement so please fix it to match the requirements 📌📌🔥🔥🚀🚀💪🏻

![image](https://user-images.githubusercontent.com/58553711/189784026-46177e24-48d1-4a65-88ff-120cc87739a7.png)